libindy/src/services/pool/mod.rs

```diff
@@ -277,7 +278,21 @@ impl PoolWorker {
                             })
                     })?;
                 }
-                &ZMQLoopAction::Timeout => {
+                ZMQLoopAction::Register(cmd_id, txn_type, parser, free) => {
+                    match self.handler {
+                        PoolWorkerHandler::TransactionHandler(ref mut th) => {
+                            th.parser_sps.insert(txn_type,
```

Comment:
We need to store "parser_sps" on a service layer. All opened pools must handle this transaction type.

Reply:
Related to https://github.com/hyperledger/indy-sdk/pull/832#discussion_r192691740
I prefer to have more flexibility here.
The case when application use more then 1 pool is something unusual and required by complex logic. For complex cases it's better to have flexible approach